lib/src/material_color_picker.dart

```diff
@@ -41,11 +41,11 @@ class MaterialColorPicker extends StatefulWidget {
 class _MaterialColorPickerState extends State<MaterialColorPicker> {
   final _defaultValue = materialColors[0];
 
-  List<ColorSwatch> _colors = materialColors;
+  List<ColorSwatch>? _colors = materialColors;
 
-  ColorSwatch _mainColor;
-  Color _shadeColor;
-  bool _isMainSelection;
+  ColorSwatch? _mainColor;
```

Comment:
Please make `_mainColor` `late`, but not nullable. It is always initialized to a non-null value in `_initSelectedValue`.